### PR TITLE
[Clang] Fix typo in ItaniumRecordLayoutBuilder API

### PR DESCRIPTION
**Pull request by Ptkyr**

### CODE REVIEW

File: clang/lib/AST/RecordLayoutBuilder.cpp [hunk 1]
```diff
@@ -818,7 +818,7 @@ class ItaniumRecordLayoutBuilder {
   void setSize(CharUnits NewSize) { Size = Context.toBits(NewSize); }
   void setSize(uint64_t NewSize) { Size = NewSize; }
 
-  CharUnits getAligment() const { return Alignment; }
+  CharUnits getAlignment() const { return Alignment; }
```
Comment (cor3ntin) [inline clang/lib/AST/RecordLayoutBuilder.cpp:821]: Is that used at all then?

Comment (zyn0217) [reply clang/lib/AST/RecordLayoutBuilder.cpp:821]: It looks like an internal class so I think the answer is no

Comment (Ptkyr) [reply clang/lib/AST/RecordLayoutBuilder.cpp:821]: Nope, just something I happened to notice when debugging something else. Not sure how well-received tiny prs like these are, but figured it was worth pointing out.

**Review by cor3ntin — APPROVED** [review]
It certainly doesn't hurt, thanks!
(but it speaks to lack of usefulness or test coverage somewhere if this is just dead code)

**Pull request merged**